src/test/java/projectzero/core/UmlClassTest.java

```diff
@@ -4,6 +4,7 @@
 import org.junit.jupiter.api.Test;
 
 public class UmlClassTest {
+
     @Test
     void testAddFieldReturnsTrueOnSuccess() {
         UmlClass umlClass = new UmlClass("class");
```

Comment:
This line is also confusing because the names UmlClass and umlClass are very similar.